src/plugins/telemetryTable/TelemetryTable.js

```diff
@@ -47,6 +47,7 @@ define([
             this.subscriptions = {};
             this.tableComposition = undefined;
             this.telemetryObjects = [];
+            this.datumCache = [];
```

Comment:
This should likely be part of a different branch and PR? I don't think it's specific to Conditionals so if it's required to go out sooner, it could.